CHANGES.rst

```diff
@@ -10,6 +10,7 @@ Bug fixes
 API changes
 -----------
 
+- Changed one final ``gala.integrate.Integrator.run`` to ``gala.integrate.Integrator.__call__``.
```

Comment:
This change doesn't need a changelog entry because it's not user-facing (just an internal change).

Reply:
Sorry about that - just removed it! :)